org.eclipse.m2e.core/src/org/eclipse/m2e/core/internal/MavenPluginActivator.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2010, 2018 Sonatype, Inc. and others.
+ * Copyright (c) 2010, 2021 Sonatype, Inc. and others.
```

Comment:
It's 2022 :)

Reply:
You git me. 😅 I did most of this last year already. Thanks for the hint, I have corrected it.